File: Kattis/Ladder.cpp
//Link:-https://open.kattis.com/problems/ladder
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    double h,v;
    cin>>h>>v;
    cout<<int(ceil(h/sin(((v*3.142)/180.0))));
    return 0;
}

File: Kattis/Ladder.java
import java.lang.*;
import java.io.*;
import java.util.*;
public class Ladder {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int h = sc.nextInt();
        int v = sc.nextInt();
        System.out.println((int)Math.ceil((double)h/Math.sin(Math.toRadians(v))));
    }
}
